src/bot.py

```diff
@@ -1,26 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+@author: AlwaysLivid
+@description: Bot that posts a tiny flower bed on Twitter every few hours.
+'''
 
 import tweepy
 import time
 import logging, os, textwrap
 from config import *
 from random import randint
-import sys
-import credentials
+
 print("""
                        TinyFlowerBeds
-
             Copyright (C) 2018-2019 AlwaysLivid
-

```

Comment:
Why were those spaces removed? That's irrelevant to the pull request, the issue and it's purpose. Please open new issues before making changes like this.